vortexasdk/endpoints/asset_tanks.py

```diff
@@ -108,4 +108,13 @@ def search(
             "term": [str(e) for e in convert_to_list(term)],
         }
 
-        return AssetTankResult(super().search(**search_params))
+        response = super().search_with_client(
```

Comment:
Can you please explain why we need `search_with_client`?

Reply:
I've explained this in another comment, and added a code comment.